static.config.js

```diff
@@ -19,6 +22,58 @@ export default {
       }),
       path: `/blog`,
     },
+    {
+      children: devops.map((dev) => ({
+        getData: () => ({
+          dev,
+        }),
+        path: `/dev/${dev.id}`,
```

Comment:
```suggestion
        path: `/${dev.id}`,
```
I would change the path to this so it becomes `/dev-ops/id` instead of `/dev-ops/dev/id`